templates/index.html

```diff
@@ -356,33 +356,36 @@
       </div>
     </div>
   </div>
-    <div class="container">
+  <br>  
```

Comment:
This is a new line. I think, your intention is to add more space between the elements.
This can be done with CSS and is the way many websites do it.

You can read about inspecting here: https://zapier.com/blog/inspect-element-tutorial/ and here is a CSS tutorial: http://learn.opentechschool.org/

You are welcome to contribute more and also apply any new knowledge you have gained. Let me know if you need anything.

Reply:
Css for making more space below of the size of one text line:
`padding-bottom: 1em;`